plugins/wazuh-core/public/services/query-manager/filters-service.ts

```diff
@@ -0,0 +1,273 @@
+import rison from 'rison-node';
+
+export enum FILTER_OPERATOR {
+  IS = 'is',
+  IS_NOT = 'is not',
+  EXISTS = 'exists',
+  DOES_NOT_EXISTS = 'does not exists',
+  IS_ONE_OF = 'is one of',
+  IS_NOT_ONE_OF = 'is not one of',
+  IS_BETWEEN = 'is between',
+  IS_NOT_BETWEEN = 'is not between',
+}
+import { TFilter } from './types';
+
+export class FiltersService {
+  static removeHiddenFilters(filters: TFilter[]) {
+    if (!filters) {
+      return filters;
+    }
+
+    return filters.filter(
+      filter => !filter.meta?.controlledBy?.startsWith('hidden-'),
+    );
+  }
+
+  static removeFilterByControlledBy(filters: TFilter[]): TFilter {
+    const controlledBy = filters.filter(
+      filter => filter.meta?.controlledBy === value,
+    );
+
+    return controlledBy;
+  }
+
+  static removeFilter(
+    filters: TFilter[],
+    field: string,
+    value: string | string[],
+  ): TFilter[] {
+    const filterIndex = filters.findIndex(f =>
```

Comment:
Perhaps a more descriptive name could be used.

Reply:
Modified in https://github.com/wazuh/wazuh-dashboard-plugins/pull/7264/commits/76648daf31cec3652856ba86c333cd967d250814